tests/sanity/test/explorer/test-api-datasets.js

```diff
@@ -19,11 +19,11 @@ const TEST_DATASET_PATTERN = 'SYS1.LINKLIB*';
 const TEST_DATASET_NAME = 'SYS1.LINKLIB';
 const TEST_DATASET_MEMBER_NAME = 'ACCOUNT';
 
+// allow self signed certs
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
+
 describe('test explorer server datasets api', function() {
   before('verify environment variables', function() {
-    // allow self signed certs
-    process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
-
```

Comment:
I think this change is not necessary for this PR and may cause unexpected test failures.